draft-ietf-cose-dilithium.md

```diff
@@ -70,57 +71,38 @@ informative:
 
 --- abstract
 
-This document describes JSON Object Signing and Encryption (JOSE) and CBOR Object Signing and Encryption (COSE) serializations for Module-Lattice-Based Digital Signature Standard (ML-DSA), which was derived from Dilithium, a Post-Quantum Cryptography (PQC) based digital signature scheme.
-
-This document does not define any new cryptography, only seralizations of existing cryptographic systems described in {{FIPS-204}}.
-
-Note to RFC Editor: This document should not proceed to AUTH48 until NIST completes paramater tuning and selection as a part of the [PQC](https://csrc.nist.gov/projects/post-quantum-cryptography) standardization process.
+This document describes JSON Object Signing and Encryption (JOSE) and CBOR Object Signing and Encryption (COSE) serializations for Module-Lattice-Based Digital Signature Standard (ML-DSA), a Post-Quantum Cryptography (PQC) digital signature scheme defined in FIPS 204.
 
 --- middle
 
 # Introduction
 
 This document describes how to use ML-DSA keys and signatures as described in {{FIPS-204}} with JOSE and COSE.
-To reduce implementation burden, the key type and thumbprint computations for ML-DSA are generic, and suitable for use with other algorithms such as SLH-DSA as described in {{-SLH-DSA}}.
 
 # Terminology
 
 {::boilerplate bcp14-tagged}
 
-# ML-DSA Algorithms
-
-The ML-DSA Signature Scheme is paramaterized to support different security levels.
-
-This document requests the registration of the following algorithms in {{-IANA.jose}}:
-
-| Name       | alg | Description
-|---
-| ML-DSA-44  | ML-DSA-44     | JSON Web Signature Algorithm for ML-DSA-44
-| ML-DSA-65  | ML-DSA-65     | JSON Web Signature Algorithm for ML-DSA-65
-| ML-DSA-87  | ML-DSA-87     | JSON Web Signature Algorithm for ML-DSA-87
-{: #jose-algorithms align="left" title="JOSE algorithms for ML-DSA"}
-
-This document requests the registration of the following algorithms in {{-IANA.cose}}:
-
-| Name       | alg | Description
-|---
-| ML-DSA-44  | TBD (requested assignment -48)     | CBOR Object Signing Algorithm for ML-DSA-44
-| ML-DSA-65  | TBD (requested assignment -49)     | CBOR Object Signing Algorithm for ML-DSA-65
-| ML-DSA-87  | TBD (requested assignment -50)     | CBOR Object Signing Algorithm for ML-DSA-87
-{: #cose-algorithms align="left" title="COSE algorithms for ML-DSA"}
+Some examples in this specification are truncated using "..." for readability.
 
 # Algorithm Key Pair Type
 
 The Algorithm Key Pair (AKP) Type is used to express Public and Private Keys for use with Algorithms.
 When this key type is used the "alg" JSON Web Key Parameter or COSE Key Common Parameter is REQUIRED.
 
+The "pub" and "priv" parameters contain bytes.
+Some "alg" values MAY specify a structure or length checks for "pub" and "priv".
+When AKP keys are expressed in JWK, "pub" and "priv" are base64url encoded.
+
```

Comment:
The "pub" and "priv" parameters contain byte strings in format specified by the "alg" value.

E.g., for ML-DSA algorithms, those are presumably ML-DSA public and private keys (which are byte strings). And for HPKE algorithms, those are presumably HPKE public and private keys for the KEM used by the cipher suite.